server/entitymanager/entitymanager_test.go

```diff
@@ -256,26 +364,124 @@ func TestGetBootstrapData(t *testing.T) {
 			BootPasswordHash: "ABCD123",
 			ServerTrustCert:  "FakeTLSCert",
 			BootConfig: &bootz.BootConfig{
-				VendorConfig: []byte("Vendor Config"),
-				OcConfig:     []byte("OC Config"),
+				VendorConfig: []byte(""),
+				OcConfig:     []byte(""),
 			},
 			Credentials: &bootz.Credentials{},
 		},
 		wantErr: false,
 	},
 	}
 
-	em := New(nil).AddChassis(bootz.BootMode_BOOT_MODE_SECURE, "Cisco", "123").AddControlCard("123A")
+	em, _ := New("")
+	em.chassisInventory[service.EntityLookup{Manufacturer: "Cisco", SerialNumber: "123"}] = &chassis
 
 	for _, test := range tests {
 		t.Run(test.desc, func(t *testing.T) {
-			got, err := em.GetBootstrapData(test.input)
+			got, err := em.GetBootstrapData(&service.EntityLookup{Manufacturer: "Cisco", SerialNumber: "123"}, test.input)
 			if (err != nil) != test.wantErr {
 				t.Errorf("GetBootstrapData(%v) err = %v, want %v", test.input, err, test.wantErr)
 			}
 			if !proto.Equal(got, test.want) {
-				t.Errorf("GetBootstrapData(%v) got %v, want %v", test.input, got, test.want)
+				t.Errorf("GetBootstrapData(%v) \n got: %v, \n want: %v", test.input, got, test.want)
+			}
+		})
+	}
+}
+
+func TestLoadConfig(t *testing.T) {
+	vendorCliConfig := `!! IOS XR Configuration 7.4.1
+!! Last configuration change at Wed Aug 18 19:55:09 2021 by cisco
+!
+username cisco
+ group root-lr
+ group cisco-support
+ password 7 01100F175804575D72
+!
+interface Loopback0
+ ipv4 address 44.44.44.44 255.255.255.255
+ ipv6 address 44::44/128
+!
+grpc
+ dscp cs4
+ port 57400
+ max-streams 128
+ max-streams-per-user 128
+ address-family dual
+ max-request-total 256
+ max-request-per-user 32
+!
+hw-module profile pbr vrf-redirect
+ssh server vrf default
+end`
+	tests := []struct {
+		desc             string
+		bootConfig       *entity.BootConfig
+		wantBootConfig   *bootz.BootConfig
+		wantVendorConfig []byte
+		wantErr          string
+	}{
+		{
+			desc: "Suceffull OC/vendor config",
```

Comment:
Check typing (here and in the below test cases).

Reply:
done